modules/google-analytics/bus/events/AddToCart.ts

```diff
@@ -0,0 +1,6 @@
+import useProduct from '#ioc/composables/useProduct'
+
+export default interface AddToCart {
```

Comment:
All of these events should not be in the google-analytics module. They should be in catalog, cart, ... modules. Some really generic events might even be in the base-commerce module (be wary of dependencies - event in base-commerce cant import anything from modules like catalog or cart because these modules depend on base-commerce and not the other way around).

Reply:
I moved it to specific modules based on event logic itself. Please, check it now.